src/resnet.py

```diff
@@ -73,6 +73,8 @@ def __init__(self,
         self.avgpool = nn.AdaptiveAvgPool2d((1, 1))
         self.fc = nn.Linear(256 * block.expansion, num_classes)
 
+        self.sigmoid = nn.Sigmoid()
+
```

Comment:
There's 2 changes in `src/resnet.py` are either necessary for this PR?

Reply:
no